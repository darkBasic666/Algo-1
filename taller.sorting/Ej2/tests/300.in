300 0
[(Raco;20),(Candelaria Sud;10),(Espinillo N.;19),(Sosa;11),(Teniente Berdina;14),(Santo Tomé;16),(Animaná;27),(Tintina;29),(Sgo. Temple;17),(Sta. Cruz;18),(Bustinza;37),(Rivadavia;2),(La Caldera;21),(Mojones S.;37),(Pilar;29),(Pasaje Las Tunas;21),(Saclantás;19),(Tostado;2),(Enrique Urien;33),(Las Cejas;32),(Gral. Lamadrid;38),(Sourigues;45),(Aldea Protestante;25),(Almafuerte;38),(Pto. Bermejo;7),(Puerta de Corral;19),(Federal;47),(Nelson;46),(Garruchos;38),(San Gregorio;26),(Las Tapias;20),(Floresta;49),(Las Lajas;32),(9;40),(La Candelaria;18),(Don Orione;23),(Aldea Elgenfeld;17),(Herradura;47),(Basavilbaso;13),(Coronel Dorrego;14),(Col. Dora;4),(Villa Gesell;43),(Campo Grande;8),(María Grande;45),(Distrito Tala;49),(Sta. Rosa de Calamuchita;11),(Esustolia;21),(Chimbas;6),(Carmen de Patagones;26),(Capitán Bermúdez;24),(Taboada;29),(Calchin Oeste;24),(Tacural;21),(Avellaneda;4),(Alcaraz S.;38),(Conscripto Bernardi;6),(Col. La Argentina;11),(Coronel Moldes;50),(Fraga;50),(Villa Tulumba;50),(Dolavón;49),(El Caín;19),(Estancia Vieja;14),(Nueve de Julio;1),(Pueblo Gral. San Martín;37),(F.Mamerto Esquiú;2),(Ayacucho;25),(Walter Moss;48),(Monigotes;22),(Angélica;40),(Los Juríes;7),(Plottier;26),(Villa R. Bermejito;17),(Quines;2),(Villa Domínico;33),(Matorrales;15),(Col. Anita;4),(Santa Lucía;25),(Rafael García;16),(Chacabuco;3),(Coghlan;30),(Justo Daract;26),(Berazategui;12),(Col. Iturraspe;50),(Esquel;21),(Liniers;10),(Río Pico;38),(Carcarañá;48),(Lamarque;5),(Lomas del Millón;19),(Cintra;23),(Ma. Luisa;46),(Agua de Oro;22),(Zenón Pereira;18),(Ricardone;26),(Col. Caroya;13),(Villa Ocampo;30),(Diamante;20),(J.J De Urquiza;19),(Coranzulli;49),(Col. Italiana;32),(Ma. Juana;27),(Romang;34),(San Ignacio;4),(Larrechea;25),(Los Quiroga;20),(Pto. Pirámides;8),(Capital;44),(Ballesteros;11),(San Martín de Las Escobas;16),(Soldini;39),(San Justo;13),(Fraile Pintado;42),(Yataití Calle;43),(Loberia;38),(Villa Tesei;14),(Gessler;43),(Ñancay;30),(Paso de La Patria;28),(Chilibroste;32),(Gaimán;33),(Sastre;22),(Col. Barón;31),(Col. Waltelina;37),(Muñiz;21),(El Timbó;16),(Alba Posse;5),(Florida;2),(Villa Flor Serrana;30),(Punta del Agua;29),(La Providencia;26),(Beccar;33),(Villa Santa Rita;40),(Sunchales;11),(San Ignacio;19),(San Benito;1),(Col. Marina;2),(Col. Oficial Nº14;18),(Marcos Paz;37),(Palacios;5),(Villa Quillinzo;37),(Cruz de Los Milagros;35),(Pampa de Los Guanacos;32),(Pueblo Uranga;28),(Ticino;19),(Morrison;36),(San Telmo;15),(Puelén;46),(Herlitzka;38),(Santa Teresita;14),(Freyre;6),(Onagolty;30),(Gral. Campos;11),(El Trapiche;16),(San Felipe;2),(Guernica;18),(El Solar;15),(Área Reserva Cinturón Ecológico;5),(Barracas;29),(Castelar;50),(Las Toninas;36),(Clorinda;2),(Buena Esperanza;1),(Copacabana;3),(Capital;1),(Portugalete;15),(Plaza de Mercedes;38),(Telsen;10),(Las Plumas;48),(Estación Lazo;35),(Frontera;43),(Rafael Castillo;17),(Coronel Baigorria;4),(San Carlos;25),(Vaqueros;18),(Guachipas;15),(San Gustavo;31),(Chipauquil;47),(Gral. Belgrano;7),(San Andrés de Giles;26),(Tapia;43),(Campo Santo;10),(Arroyo Ceibal;28),(Calte. Cordero;31),(Costa Sacate;25),(El Bordo;44),(Col. Raquel;46),(Gral. Almada;29),(Saavedra;30),(Padre Monti;26),(La Brava;44),(Fray Luis Beltrán;18),(San Eduardo;10),(El Piquete;26),(Aviá Teraí;41),(El Potrero;3),(Las Juntas;41),(Campana;26),(Lavalle;48),(San Fabián;6),(Resistencia;14),(El Quebrachal;16),(Pinar del Sol;21),(Santa Rosa de Leales;50),(Chañar Viejo;23),(Las Toscas;45),(Distrito Sauce;10),(Merlo;24),(Campo Viera;41),(Franck;15),(Villa Udaondo;40),(Gral. Lamadrid;13),(Gral. Acha;42),(Villa Sarmiento;42),(Pavón Arriba;42),(Villa Ballester;8),(Icaño;39),(Santa Clara;45),(Lavalle;28),(Santa Elena;11),(El Tala;13),(Guaviravi;41),(Plaza Clucellas;27),(Cortaderas;29),(Los Gómez;26),(Gral. Alvear;43),(Col. Tirolesa;16),(Villa Curí Leuvu;13),(San José de Las Salinas;44),(Los Altos;43),(Tacuarendí;27),(Col. Gral. Roca;42),(Sta. Ana;48),(Rosario de Lerma;21),(Cerro Corá;21),(18;19),(Gral. Ramírez;19),(Villa Allende;11),(Balvanera;5),(San Bernardo (S.J.);11),(Sierra Grande;11),(Silva;9),(Suco;29),(Guañacos;5),(25 de Mayo;25),(Col. Ensayo;24),(Las Peñas;48),(Virrey del Pino;20),(Ing. Jacobacci;47),(San Marcial;30),(Melincué;7),(La Playosa;25),(Luis Palacios;11),(Santa Ana;2),(Carreras;35),(Col. Alemana;22),(Mte. Maíz;50),(San Esteban;10),(Garuhapé;32),(San Fernando del Valle;4),(Gral. Deheza;32),(Pto. Madryn;50),(Saldán;37),(Los Catutos;13),(Isla De Cañas;12),(Ñanducita;11),(Ubajay;8),(San Martín;2),(Cañada Rica;15),(Cangrejillos;17),(San Lorenzo;40),(La Paquita;48),(Sta. Teresa;44),(Villa San Esteban;42),(La Verde;6),(Guaraní;36),(Gdor. Roca;42),(Los Cisnes;19),(Gdor. Echagüe;14),(Santurce;46),(Pueblo Muñoz;16),(Villa Rossi;18),(Pastor Britos;10),(Tancacha;21),(Loma Hermosa;5),(Riachuelo;42),(Garza;15),(Capitan Gral B. O´Higgins;19),(Del Viso;12),(Añelo;1),(San Bernardo;8),(Belgrano;44),(Los Talares;24),(Las Tunas;14),(Vera y Pintado;6),(El Aguilar;2),(Olivares San Nicolás;12),(Don Cristóbal 1ª Sección;35),(San Víctor;5),(Caucete;7)]
