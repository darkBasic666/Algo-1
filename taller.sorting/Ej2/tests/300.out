300 0
[(Añelo;1),(Buena Esperanza;1),(Capital;1),(Nueve de Julio;1),(San Benito;1),(Clorinda;2),(Col. Marina;2),(El Aguilar;2),(F.Mamerto Esquiú;2),(Florida;2),(Quines;2),(Rivadavia;2),(San Felipe;2),(San Martín;2),(Santa Ana;2),(Tostado;2),(Chacabuco;3),(Copacabana;3),(El Potrero;3),(Avellaneda;4),(Col. Anita;4),(Col. Dora;4),(Coronel Baigorria;4),(San Fernando del Valle;4),(San Ignacio;4),(Alba Posse;5),(Balvanera;5),(Guañacos;5),(Lamarque;5),(Loma Hermosa;5),(Palacios;5),(San Víctor;5),(Área Reserva Cinturón Ecológico;5),(Chimbas;6),(Conscripto Bernardi;6),(Freyre;6),(La Verde;6),(San Fabián;6),(Vera y Pintado;6),(Caucete;7),(Gral. Belgrano;7),(Los Juríes;7),(Melincué;7),(Pto. Bermejo;7),(Campo Grande;8),(Pto. Pirámides;8),(San Bernardo;8),(Ubajay;8),(Villa Ballester;8),(Silva;9),(Campo Santo;10),(Candelaria Sud;10),(Distrito Sauce;10),(Liniers;10),(Pastor Britos;10),(San Eduardo;10),(San Esteban;10),(Telsen;10),(Ballesteros;11),(Col. La Argentina;11),(Gral. Campos;11),(Luis Palacios;11),(San Bernardo (S.J.);11),(Santa Elena;11),(Sierra Grande;11),(Sosa;11),(Sta. Rosa de Calamuchita;11),(Sunchales;11),(Villa Allende;11),(Ñanducita;11),(Berazategui;12),(Del Viso;12),(Isla De Cañas;12),(Olivares San Nicolás;12),(Basavilbaso;13),(Col. Caroya;13),(El Tala;13),(Gral. Lamadrid;13),(Los Catutos;13),(San Justo;13),(Villa Curí Leuvu;13),(Coronel Dorrego;14),(Estancia Vieja;14),(Gdor. Echagüe;14),(Las Tunas;14),(Resistencia;14),(Santa Teresita;14),(Teniente Berdina;14),(Villa Tesei;14),(Cañada Rica;15),(El Solar;15),(Franck;15),(Garza;15),(Guachipas;15),(Matorrales;15),(Portugalete;15),(San Telmo;15),(Col. Tirolesa;16),(El Quebrachal;16),(El Timbó;16),(El Trapiche;16),(Pueblo Muñoz;16),(Rafael García;16),(San Martín de Las Escobas;16),(Santo Tomé;16),(Aldea Elgenfeld;17),(Cangrejillos;17),(Rafael Castillo;17),(Sgo. Temple;17),(Villa R. Bermejito;17),(Col. Oficial Nº14;18),(Fray Luis Beltrán;18),(Guernica;18),(La Candelaria;18),(Sta. Cruz;18),(Vaqueros;18),(Villa Rossi;18),(Zenón Pereira;18),(18;19),(Capitan Gral B. O´Higgins;19),(El Caín;19),(Espinillo N.;19),(Gral. Ramírez;19),(J.J De Urquiza;19),(Lomas del Millón;19),(Los Cisnes;19),(Puerta de Corral;19),(Saclantás;19),(San Ignacio;19),(Ticino;19),(Diamante;20),(Las Tapias;20),(Los Quiroga;20),(Raco;20),(Virrey del Pino;20),(Cerro Corá;21),(Esquel;21),(Esustolia;21),(La Caldera;21),(Muñiz;21),(Pasaje Las Tunas;21),(Pinar del Sol;21),(Rosario de Lerma;21),(Tacural;21),(Tancacha;21),(Agua de Oro;22),(Col. Alemana;22),(Monigotes;22),(Sastre;22),(Chañar Viejo;23),(Cintra;23),(Don Orione;23),(Calchin Oeste;24),(Capitán Bermúdez;24),(Col. Ensayo;24),(Los Talares;24),(Merlo;24),(25 de Mayo;25),(Aldea Protestante;25),(Ayacucho;25),(Costa Sacate;25),(La Playosa;25),(Larrechea;25),(San Carlos;25),(Santa Lucía;25),(Campana;26),(Carmen de Patagones;26),(El Piquete;26),(Justo Daract;26),(La Providencia;26),(Los Gómez;26),(Padre Monti;26),(Plottier;26),(Ricardone;26),(San Andrés de Giles;26),(San Gregorio;26),(Animaná;27),(Ma. Juana;27),(Plaza Clucellas;27),(Tacuarendí;27),(Arroyo Ceibal;28),(Lavalle;28),(Paso de La Patria;28),(Pueblo Uranga;28),(Barracas;29),(Cortaderas;29),(Gral. Almada;29),(Pilar;29),(Punta del Agua;29),(Suco;29),(Taboada;29),(Tintina;29),(Coghlan;30),(Onagolty;30),(Saavedra;30),(San Marcial;30),(Villa Flor Serrana;30),(Villa Ocampo;30),(Ñancay;30),(Calte. Cordero;31),(Col. Barón;31),(San Gustavo;31),(Chilibroste;32),(Col. Italiana;32),(Garuhapé;32),(Gral. Deheza;32),(Las Cejas;32),(Las Lajas;32),(Pampa de Los Guanacos;32),(Beccar;33),(Enrique Urien;33),(Gaimán;33),(Villa Domínico;33),(Romang;34),(Carreras;35),(Cruz de Los Milagros;35),(Don Cristóbal 1ª Sección;35),(Estación Lazo;35),(Guaraní;36),(Las Toninas;36),(Morrison;36),(Bustinza;37),(Col. Waltelina;37),(Marcos Paz;37),(Mojones S.;37),(Pueblo Gral. San Martín;37),(Saldán;37),(Villa Quillinzo;37),(Alcaraz S.;38),(Almafuerte;38),(Garruchos;38),(Gral. Lamadrid;38),(Herlitzka;38),(Loberia;38),(Plaza de Mercedes;38),(Río Pico;38),(Icaño;39),(Soldini;39),(9;40),(Angélica;40),(San Lorenzo;40),(Villa Santa Rita;40),(Villa Udaondo;40),(Aviá Teraí;41),(Campo Viera;41),(Guaviravi;41),(Las Juntas;41),(Col. Gral. Roca;42),(Fraile Pintado;42),(Gdor. Roca;42),(Gral. Acha;42),(Pavón Arriba;42),(Riachuelo;42),(Villa San Esteban;42),(Villa Sarmiento;42),(Frontera;43),(Gessler;43),(Gral. Alvear;43),(Los Altos;43),(Tapia;43),(Villa Gesell;43),(Yataití Calle;43),(Belgrano;44),(Capital;44),(El Bordo;44),(La Brava;44),(San José de Las Salinas;44),(Sta. Teresa;44),(Las Toscas;45),(María Grande;45),(Santa Clara;45),(Sourigues;45),(Col. Raquel;46),(Ma. Luisa;46),(Nelson;46),(Puelén;46),(Santurce;46),(Chipauquil;47),(Federal;47),(Herradura;47),(Ing. Jacobacci;47),(Carcarañá;48),(La Paquita;48),(Las Peñas;48),(Las Plumas;48),(Lavalle;48),(Sta. Ana;48),(Walter Moss;48),(Coranzulli;49),(Distrito Tala;49),(Dolavón;49),(Floresta;49),(Castelar;50),(Col. Iturraspe;50),(Coronel Moldes;50),(Fraga;50),(Mte. Maíz;50),(Pto. Madryn;50),(Santa Rosa de Leales;50),(Villa Tulumba;50)]
